src/ppx/grfx/dx12/dx12_descriptor.cpp

```diff
@@ -313,6 +313,22 @@ Result DescriptorSet::UpdateDescriptors(uint32_t writeCount, const grfx::WriteDe
             } break;
 
             case grfx::DESCRIPTOR_TYPE_STORAGE_BUFFER: {
```

Comment:
it is mentioned in the description of this change, but i still feel that it is confusing to have use the type Descriptor_type_storage_buffer for raw buffer since the comment in grfx_enums.h also doesn't indicate this is a raw buffer. Maybe better with a new type?
```
DESCRIPTOR_TYPE_STORAGE_BUFFER         = 8,  // RW structured buffer object
```

Reply:
I will rename it to RAW_STORAGE_BUFFER. There's no use apart from this, so we're safe to rename.